docs/proposals/README.md

```diff
@@ -0,0 +1,38 @@
+# Gardener Enhancement Proposal (GEP)
+
+Changes to the Gardener code base are often incorporated directly via pull requests which either themselves contain a description about the motivation and scope of a change or a linked Github issue does.
+
+If a perspective feature has a bigger extent, requires the involvement or several parties or more discussion is needed before the actual implementation starts, you may consider filing a pull request with a Gardener Enhancement Proposal (GEP) first.
+
+GEPs are a measure to propose a change or to add a feature to Gardener, helps you to describe the change(s) conceptionally and to list the steps that are necessary to reach this goal. It helps the Gardener maintainers as well as the community to understand the motivation and scope around your proposed change(s) and encourages their contribution to discussions and future pull requests. If you are familiar with the Kubernetes community, GEPs are analogue to Kubernetes Enhancement Proposals ([KEPs]( https://github.com/kubernetes/enhancements/tree/master/keps
+)).
+
+## Reasons for a GEP
+
+You may consider filing a GEP for the following reasons:
+-	A Gardener architectural change is intended / necessary
+-	Major changes to the Gardener code base
+-	A phased implementation approach is expected because of the widespread scope of the change
+-	Your proposed changes may be controversial
+
+We encourage you to take a look at already merged [GEPs]( https://github.com/gardener/gardener/tree/master/docs/proposals) since they give you a sense of what a typical GEP comprises.
+
+## Before creating a GEP
+
+It is recommended to discuss and outline the motivation of your prospective GEP as a draft with the community before you take the investment of creating the actual GEP. This early briefing supports the understanding for the broad community and leads to a fast feedback for your proposal from the respective experts in the community.
+An appropriate format for this may be the regular [Gardener community meeting]( https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#weekly-meeting
+).
+
+## How to file a GEP
+
+GEPs should be created as Markdown `.md` files and are submitted through a Github pull request to their current home in [docs/proposals](https://github.com/gardener/gardener/tree/master/docs/proposals). Please use the provided [template](./00-template.md) or follow the structure of existing [GEPs]( https://github.com/gardener/gardener/tree/master/docs/proposals) which makes reviewing easier and faster.
+
+If not already done, please present your GEP in the [regular community meeting]( https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#weekly-meeting) to brief the community about your proposal (we strive for personal communication :) ). Also consider that this may be an important step to raise awareness and understanding for everyone involved.
+
+## GEP Process
```

Comment:
Would it make sense to also add some guideline on the actual implementation process of such larger changes?
E.g. we could state, that for bigger changes it is probably good to be introduced behind a `FeatureGate` (and should then be observed for some time and graduated after enough validation).

Reply:
Imo, this is not very related to GEPs in general and can be discussed case-by-case in the respective PR.

Reply:
Yep, sounds good.